docs/_source/getting_started/installation/deployments/cloud_providers.md

```diff
@@ -50,7 +50,16 @@ docker-machine create --driver amazonec2 \
 argilla-aws
 ```
 
-Available ami depends on region. The provided ami is available for eu-west regions
+Available ami depends on region. The provided ami is available for eu-west regions and is an ubuntu-16.04-server image. 
```

Comment:
Since here's the first occurrence of `ami`, I'll update it to the following:

```suggestion
Available Amazon Machine Instance (AMI) depends on region. The provided AMI is available for eu-west regions and is an ubuntu-16.04-server image.
```